<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceConfigWidgetBase</class>
 <widget class="QWidget" name="DeviceConfigWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QFrame" name="Frame3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32567</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <family>Nimbus Sans l [Urw]</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Device Settings</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="Spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>181</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer106_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QGroupBox" name="cashbox_groupbox">
       <property name="title">
        <string>Cash Drawer</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="cashbox_icon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>image0</pixmap>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Drawer Type</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Device</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QComboBox" name="cashbox_type_combobox">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>RS-232 Serial</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ESC/POS Printer</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cashbox_device_combobox">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>/dev/ttyS0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyS3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/lp0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/lp1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>/dev/ttyUSB3</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer106_2_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="printer_groupbox">
       <property name="title">
        <string>Printer</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer10_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="printer_icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>image1</pixmap>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="printer_chars_groupbox">
          <property name="title">
           <string>Line Size</string>
          </property>
          <property name="exclusive" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QRadioButton" name="printer_length_42_radiobutton">
             <property name="text">
              <string>42 Characters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="printer_length_40_radiobutton">
             <property name="text">
              <string>40 Characters</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel2_3">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Printer Type</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2_2_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Device</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QComboBox" name="printer_type_combobox">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Local</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Remote (IPP)</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="spacer15">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>271</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QComboBox" name="printer_device_combobox">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>32767</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>/dev/lp0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/lp1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB3</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="printer_lineedit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer9_2_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer9_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer106_2_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="kitchen_groupbox">
       <property name="title">
        <string>Kitchen Printer</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer10_2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="kitchen_icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>image1</pixmap>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="kitchen_chars_groupbox">
          <property name="title">
           <string>Line Size</string>
          </property>
          <property name="exclusive" stdset="0">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QRadioButton" name="kitchen_length_42_radiobutton">
             <property name="text">
              <string>42 Characters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="kitchen_length_40_radiobutton">
             <property name="text">
              <string>40 Characters</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel2_3_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Printer Type</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2_2_2_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Device</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QComboBox" name="kitchen_type_combobox">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Local</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Remote (IPP)</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="spacer16">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>261</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QComboBox" name="kitchen_device_combobox">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>32767</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>/dev/lp0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/lp1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyS3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>/dev/ttyUSB3</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="kitchen_lineedit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer9_2_2_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer9_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer106">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line8">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer103">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>511</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
